packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -64,93 +68,79 @@ export interface MeetingAdapterMeetingManagement
-  off(event: 'meetingEnded', listener: CallEndedListener): void;
   off(event: 'error', listener: (e: Error) => void): void;
 
   // Call subscriptions
+  on(event: 'callEnded', listener: CallEndedListener): void;
   on(event: 'isMutedChanged', listener: IsMutedChangedListener): void;
   on(event: 'callIdChanged', listener: CallIdChangedListener): void;
   on(event: 'isLocalScreenSharingActiveChanged', listener: IsLocalScreenSharingActiveChangedListener): void;
   on(event: 'displayNameChanged', listener: DisplayNameChangedListener): void;
   on(event: 'isSpeakingChanged', listener: IsSpeakingChangedListener): void;
+  on(event: 'callParticipantsJoined', listener: ParticipantsJoinedListener): void;
+  on(event: 'callParticipantsLeft', listener: ParticipantsLeftListener): void;
 
+  off(event: 'callEnded', listener: CallEndedListener): void;
   off(event: 'isMutedChanged', listener: IsMutedChangedListener): void;
   off(event: 'callIdChanged', listener: CallIdChangedListener): void;
   off(event: 'isLocalScreenSharingActiveChanged', listener: IsLocalScreenSharingActiveChangedListener): void;
   off(event: 'displayNameChanged', listener: DisplayNameChangedListener): void;
   off(event: 'isSpeakingChanged', listener: IsSpeakingChangedListener): void;
+  off(event: 'callParticipantsJoined', listener: ParticipantsJoinedListener): void;
+  off(event: 'callParticipantsLeft', listener: ParticipantsLeftListener): void;
 
   // Chat subscriptions
   on(event: 'messageReceived', listener: MessageReceivedListener): void;
   on(event: 'messageSent', listener: MessageSentListener): void;
   on(event: 'messageRead', listener: MessageReadListener): void;
+  on(event: 'chatParticipantsAdded', listener: ParticipantsAddedListener): void;
+  on(event: 'chatParticipantsRemoved', listener: ParticipantsRemovedListener): void;
 
   off(event: 'messageReceived', listener: MessageReceivedListener): void;
   off(event: 'messageSent', listener: MessageSentListener): void;
   off(event: 'messageRead', listener: MessageReadListener): void;
+  off(event: 'chatParticipantsAdded', listener: ParticipantsAddedListener): void;
+  off(event: 'chatParticipantsRemoved', listener: ParticipantsRemovedListener): void;
 }
 
 /**
- * Meeting Composite Adapter interface.
+ * {@link CallAndChatComposite} Adapter interface.
  * @beta
  */
-export interface MeetingAdapter
-  extends MeetingAdapterMeetingManagement,
-    AdapterState<MeetingAdapterState>,
+export interface CallAndChatAdapter
+  extends CallAndChatAdapterManagement,
+    AdapterState<CallAndChatAdapterState>,
     Disposable,
-    MeetingAdapterSubscriptions {}
+    CallAndChatAdapterSubscriptions {}
 
 /**
- * Events fired off by the Meeting Adapter
+ * Events fired off by the {@link ChatAndCallAdapter}
  * @beta
  */
-export type MeetingEvent =
-  | 'participantsJoined'
-  | 'participantsLeft'
-  | 'meetingEnded'
+export type CallAndChatEvent =
+  | 'error'
```

Comment:
Should this be `callError` and `chatError` as well?

Reply:
Updated to callError and chatError